Tests/Functional/Controller/ConnectControllerTest.php

```diff
@@ -133,7 +129,10 @@ public function testConnectService(): void
         $this->assertSame('Successfully connected the account "foo"!', $crawler->filter('h3')->text(), $response->getContent());
     }
 
-    private function logIn(Client $client, SessionInterface $session): void
+    /**
+     * @param Client|KernelBrowser $client
+     */
+    private function logIn($client, SessionInterface $session): void
```

Comment:
we can typehint against `KernelBrowser` since its available on Symfony 4.4 as well and its the minimum required version, or?

Reply:
Will check, for some reason (probably too low acceptable dependency) I was not seeing this class while testing.